packages/framework/block-std/src/scope/block-std-scope.ts

```diff
@@ -26,6 +27,8 @@ export class BlockStdScope {
 
   readonly event: UIEventDispatcher;
 
+  readonly gfx: GfxController;
```

Comment:
The gfx block (e.g., note block, image block) may need to access gfx-related APIs or something similar. I think these blocks should rely on a standard entry rather than a specific type of service or block.

We can make the `std.gfx` an extended field using [merging interfaces](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces). This way, the default std does not include the gfx by default unless you explicitly import the `std/gfx`.

Reply:
Regarding https://github.com/toeverything/blocksuite/pull/8094. I think edgeless can also have similar extension entry point to block service.